cylc/flow/network/ssh_client.py

```diff
@@ -69,6 +93,18 @@ def send_request(self, command, args=None, timeout=None):
         Returns:
             object: Deserialized output from function called.
         """
+        loop = asyncio.new_event_loop()
+        task = loop.create_task(
+            self.async_request(command, args, timeout))
+        loop.run_until_complete(task)
+        loop.close()
```

Comment:
Dammit asyncio is annoying, where's the "just do it" interface?

Reply:
I think you can skip the creat_task stage and `loop.run_until_complete(the_thing_you_want_to_run())` but I think this is fine.

Reply:
> Dammit asyncio is annoying, where's the "just do it" interface?

I think this is it...
```
task = asyncio.run(
            self.async_request(command, args, timeout))
```

Reply:
Except that it can raise `RuntimeError`.

Reply:
Perhaps a solution in the style of: https://stackoverflow.com/a/61331974

Reply:
Or stick with the original that opens a new loop and closes it manually.

Reply:
> Or stick with the original 

I think that's fine.

We have an existing issue with the RuntimeError problem in the functional tests, may need a solution like the one you linked to.

Reply:
FYI: The way I've done async/sync interfaces elsewhere is:

```
def async_whatever(...):
    ...

def whatever(*args, **kwargs):
    # This will work, but only if your code is not called by async code.
    # This assumption can be broken in the unit/integration tests.
    asyncio.run(async_whatever(*args, **kwargs))
```